### ISSUE TITLE
Better documentation for dnsmasq

### ISSUE THREAD

**Issue by fredrike** [alardus/portaller#20]
It would be great if the list here http://portaller.com/setup#dnsmasq would be more complete on which domains to add to dnsmasq.conf (which you are supporting)

Love the service!

**Comment by fredrike** [comment]
Just as a note, to get netflix workin on my iPad I had to add the following to dnsmasq.conf:
```
server=/netflix.com/107.170.15.247
server=/netflix.com/77.88.8.8
server=/e.akamai.net/107.170.15.247
server=/nflxext.com/107.170.15.247
server=/nflximg.com/107.170.15.247
```

I'm not sure what made it work though..

**Comment by alardus** [comment]
Just updated list of domains, check it out http://portaller.com/setup#dnsmasq. Would you please set it up instead your current list of domains and check Netflix on iPad?

**Comment by fredrike** [comment]
Thank you!

Tested it and it works perfectly (both on my mac and on my iPad). Will test on my Chromecast later tonight (have configured my router with "intercept DNS port" as shown below.

    iptables -t nat -A PREROUTING -i br0 -p udp --dport 53 -j DNAT --to $(nvram get lan_ipaddr)
    iptables -t nat -A PREROUTING -i br0 -p tcp --dport 53 -j DNAT --to $(nvram get lan_ipaddr)

Again thank you for great service (would donate if you had a PayPal link).

**Comment by alardus** [comment]
Fixed at 7b36c950a509dfa05565627532253540d03072b5. Appreciate your support!

**Comment by fredrike** [comment]
It works fine on my Chromecast, Android phones and iPads but not on my Sony BD. This is my dnsmasq log (when booting and trying to access Netflix):

```
Mar 30 17:05:55 unknown daemon.info dnsmasq-dhcp[30412]: DHCPOFFER(br0) 192.168.1.34 3c:07:71:cd:3d:XX 
Mar 30 17:05:55 unknown daemon.info dnsmasq-dhcp[30412]: DHCPREQUEST(br0) 192.168.1.34 3c:07:71:cd:3d:XX 
Mar 30 17:05:55 unknown daemon.info dnsmasq-dhcp[30412]: DHCPACK(br0) 192.168.1.34 3c:07:71:cd:3d:XX 
Mar 30 17:05:55 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:05:55 unknown daemon.info dnsmasq[30412]: query[AAAA] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:05:55 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:05:56 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:05:56 unknown daemon.info dnsmasq[30412]: query[AAAA] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:05:56 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:05:57 unknown daemon.info dnsmasq[30412]: query[A] m.root-servers.net from 192.168.1.34
Mar 30 17:05:58 unknown daemon.info dnsmasq[30412]: query[A] ssm.internet.sony.tv from 192.168.1.34
Mar 30 17:05:58 unknown daemon.info dnsmasq[30412]: query[AAAA] ssm.internet.sony.tv from 192.168.1.34
Mar 30 17:05:58 unknown daemon.info dnsmasq[30412]: query[A] ssm.internet.sony.tv from 192.168.1.34
Mar 30 17:05:58 unknown daemon.info dnsmasq[30412]: query[A] blu-ray.update.sony.net from 192.168.1.34
Mar 30 17:05:58 unknown daemon.info dnsmasq[30412]: query[AAAA] blu-ray.update.sony.net from 192.168.1.34
Mar 30 17:05:58 unknown daemon.info dnsmasq[30412]: query[A] blu-ray.update.sony.net from 192.168.1.34
Mar 30 17:05:59 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:05:59 unknown daemon.info dnsmasq[30412]: query[AAAA] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:05:59 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:06:00 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:06:00 unknown daemon.info dnsmasq[30412]: query[AAAA] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:06:00 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:06:02 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:06:02 unknown daemon.info dnsmasq[30412]: query[AAAA] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:06:02 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:06:03 unknown daemon.info dnsmasq[30412]: query[A] call.me.sel.sony.com from 192.168.1.34
Mar 30 17:06:03 unknown daemon.info dnsmasq[30412]: query[AAAA] call.me.sel.sony.com from 192.168.1.34
Mar 30 17:06:03 unknown daemon.info dnsmasq[30412]: query[A] call.me.sel.sony.com from 192.168.1.34
Mar 30 17:06:36 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:06:36 unknown daemon.info dnsmasq[30412]: query[AAAA] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:06:36 unknown daemon.info dnsmasq[30412]: query[A] ssm1.internet.sony.tv from 192.168.1.34
Mar 30 17:06:46 unknown daemon.info dnsmasq[30412]: query[A] uiboot.netflix.com from 192.168.1.34
Mar 30 17:06:46 unknown daemon.info dnsmasq[30412]: query[AAAA] uiboot.netflix.com from 192.168.1.34
Mar 30 17:06:49 unknown daemon.info dnsmasq[30412]: query[A] nrdp.nccp.netflix.com from 192.168.1.34
Mar 30 17:06:49 unknown daemon.info dnsmasq[30412]: query[AAAA] nrdp.nccp.netflix.com from 192.168.1.34
Mar 30 17:06:49 unknown daemon.info dnsmasq[30412]: query[A] nrdp.nccp.netflix.com from 192.168.1.34
```

**Comment by alardus** [comment]
Looks similar to https://github.com/alardus/portaller/issues/17 - I had same symptoms on LG Smart TV. Problem is that Sony Netflix client does not support SNI. To spoof it this three domains must be hosted at individual IP: uiboot.netflix.com, secure.netflix.com and api-global.netflix.com.

I made this experiment and it was successful, but need to have 3 additional VPS (to get 3 new IP) and it is getting a bit expensive. So I turned it off and looking for a way to have additional IP's or VPS at low price.
